File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/linqexamples44.cs
// <snippetlinqexamples44>


 using (ServiceContext svcContext = new ServiceContext(_serviceProxy))
 {
  var query_retrieve1 = from c in svcContext.ContactSet
                        join a in svcContext.AccountSet
                        on c.ContactId equals a.PrimaryContactId.Id
                        where c.ContactId != _contactId1
                        select new { Contact = c, Account = a };
  foreach (var c in query_retrieve1)
  {
   System.Console.WriteLine("Acct: " + c.Account.Name +
    "\t\t" + "Contact: " + c.Contact.FullName);
  }
 }

// </snippetlinqexamples44>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/queryvb/vb/linqexamples44.vb
' <snippetlinqexamples44>


 Using svcContext As New ServiceContext(_serviceProxy)
  Dim query_retrieve1 = From c In svcContext.ContactSet _
                        Join a In svcContext.AccountSet _
                        On c.ContactId Equals a.PrimaryContactId.Id _
                        Where Not c.ContactId.Value.Equals(_contactId1) _
                        Select New With {Key .Contact = c,
                                         Key .Account = a}
  For Each c In query_retrieve1
   Console.WriteLine("Acct: " &amp; c.Account.Name &amp; vbTab &amp; vbTab _
                     &amp; "Contact: " &amp; c.Contact.FullName)
  Next c
 End Using

' </snippetlinqexamples44>